web/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "osm2streets-web",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "wasm": "wasm-pack build --dev --target web ../osm2streets-js",
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "check": "svelte-check --tsconfig ./tsconfig.json",
+    "fmt": "npx prettier --write *.html src/**",
+    "list-tests": "./public/list_tests.sh"
```

Comment:
Have to manually run this when adding new tests. There could still be nicer solutions here...